src/html-requires-loader.ts

```diff
@@ -9,6 +9,8 @@ function loader(this: Webpack.Loader, content: string) {
       const attrs = loader.attributes[tag];
       return attrs && attrs.includes(attr);
     })
+    // Ignore values that contain interpolated values
+    .filter(attr => !/(^|[^\\])\$\{/.test(attr.value))
```

Comment:
@jods4 Perhaps we could emit warnings or at least inform the user in some other way about all the values which match such a scenario and additionally inform them about a way to mitigate the issue? It might be hard to debug this problem for new users, scratching their heads why interpolation does not work.